tests/libcxx/CMakeLists.txt

```diff
@@ -19,5 +19,12 @@ endif()
 
 foreach(testcase ${alltests})
     get_testcase_name(${testcase} name "../../3rdparty/libcxx/libcxx/test/")
-	add_enclave_test(tests/libcxxtest-${name} ./host libcxx_host ./enc libcxxtest-${name}_enc)
+
+    if ("${name}" MATCHES "cons_default_throws_bad_alloc.pass" OR "${name}" MATCHES "allocator_allocator.members_construct.pass")
```

Comment:
Please add a comment indicating why this exception is used here and in enc/CMakeLists.txt:
```
# the allocations are entirely optimized out by Clang in these tests and are excluded from Clang release builds
```